test/test_replaygain.py

```diff
@@ -55,10 +59,28 @@ def reset_replaygain(item):
     item['rg_album_gain'] = None
     item.write()
     item.store()
+    item.store()
+    item.store()
 
 
-class ReplayGainCliTestBase(TestHelper):
+def _store_retry_once(self, item):
+    """Helper method to retry item.store() once in case
+    of a sqlite3.OperationalError exception.
 
+    :param self: `ReplayGainPlugin` instance
+    :param item: a library item to store
+    """
+    try:
+        item.store()
+    except OperationalError:
+        # test_replaygain.py :memory: library can fail with
+        #    `sqlite3.OperationalError: no such table: items`
+        # but the second attempt succeeds
+        item.store()
+
+
+@patch.object(ReplayGainPlugin, '_store', _store_retry_once)
```

Comment:
As far as I can see there's no way to `try/except` on a higher level; the "false alarm" happens at *every* item, so it looks like we still need to ignore it for every item...